app/controllers/hq/partners_controller.rb

```diff
@@ -5,4 +5,48 @@ class Hq::PartnersController < ApplicationController
   def index
     @partners= Partner.all
   end
+
+  def edit
+    load_partner
+    load_associations
+  end
+
+  def update
+    load_partner
+    load_associations
```

Comment:
Do you need to `load_associations` here for `render 'edit'`?

Do we want to try following Sandi Metz's rule that a controller should not instantiate more than one object? And adopt e.g. [this solution](http://robots.thoughtbot.com/sandi-metz-rules-for-developers) (bottom of page).